README.md

```diff
@@ -1,28 +1,66 @@
 # piponic
-IoT Monitoring and Control for Aquaponic and Hydroponic Systems Using Raspberry Pi Zero W 
+IoT Monitoring and Control for Aquaponic and Hydroponic Systems Using Raspberry Pi.
 
 ## Install
 
+To install this project on the Raspberry Pi and have it run on boot,
+please run the following:
+
 ```
-sudo ./setup.sh
+sudo ./install.sh <DEVICE-ID> <REGISTRY-ID> <PROJECT-ID>
 ```
 
-Please contact Jayden for questions about connecting to Google Cloud IoT.
+Note that the parameters are as follows:
+
+- `<DEVICE-ID>` string name to call the device (you choose)
+- `<REGISTRY-ID>` string name of the Google IoT Registry (can be new or pre-existing)
+- `<PROJECT-ID>` string name of the Google Cloud Project
 
-## Run
+Example:
 
 ```
-python3 cloudiot_pubsub_example_mqtt_device.py \               
-    --project_id=piponics \
-    --registry_id=RaspberryPis \
-    --device_id=<DEVICE-ID> \
-    --private_key_file=/home/pi/piponic/rsa_private.pem \
-    --algorithm=RS256
+sudo ./install.sh JaydenPi RaspberryPis piponics 
 ```
```

Comment:
It would be helpful here to have a reminder to copy the device's public key into the google-iot core function at this point (I always forget this after rerunning setup.sh). Is there a function that can do this or would that be a security risk?

Reply:
That is to say it didn't connect automatically for me as it suggests it might down below..

Reply:
That's weird. There is that functionality to automatically add the public key into the Google Cloud IoT. You can see this on line 104 of `install.sh`. 

Can you give me the exact steps you did when you ran the script, including:

- What was the exact command you entered?
- Did you just checkout this branch and run the script? 
- When did it start connecting if not automatically? After you rebooted the RPi?

Reply:
I did the following in this order
 - git checkout launchScriptOnBoot
 - git pull origin launchScriptOnBoot
 - sudo ./install.sh CarsonPi RaspberryPis piponics

And then it was running, but it was failing to connect. The IotCore said that no device was connected. So I added it's public key to the IoTCore lib. Then I believe I restarted the process (after checking out the status and journalctl of the piponic service) with 
 - sudo systemctl restart piponic.service

And then, if I'm not mistaken, it was connected!

Reply:
Hmm, I'm not sure why that happened. A few more questions:

- Did the connection work right before switching from the `main` branch to my branch?  
- Did the same `public_key.pem` file exist that you were using before when you ran the script?

If there's no clear answer, let's try running the script again to see if we can reproduce the issue.